freqtrade/freqtradebot.py

```diff
@@ -414,10 +414,12 @@ def create_trade(self, pair: str) -> bool:
             if ((bid_check_dom.get('enabled', False)) and
                     (bid_check_dom.get('bids_to_ask_delta', 0) > 0)):
                 if self._check_depth_of_market_buy(pair, bid_check_dom):
+                    logger.info(f'Executed Buy for {pair}.')
```

Comment:
if it's before `execute_buy`, then it probably should state "Executing..." (not yet executed, may be confusing along with debug messages from `execute_buy`)

Reply:
Problem here is there's a return statement after this line, so I couldn't move it after.

Reply:
I think this function needs further refactoring in the future.

Reply:
if you wanna leave that here for now it's ok - but it's a time question ...

this is saying "executed" - but it's really just about to execute ... so if execute_buy logs anything (maybe an exception ..?) - the bot's prior log-message (line above the exception, most likely) is highly irritating in that case - as it just told me you did that already...

Think about it
```
Executed buy for pair ETH/BTC.
Exception while buying ETH/BTC, Error XYZ on exchange whatever.....
```

This makes the first line a lie of some sort.

Reply:
Fixed to "Executing ..." for buys and sells.